.github/workflows/build-assets-compilation.yml

```diff
@@ -156,6 +157,8 @@ jobs:
           echo "ENCORE_ENV=production" >> $GITHUB_ENV
 
       - name: Compile assets
+        env:
+          COMPOSER_AUTH: '${{ secrets.COMPOSER_AUTH_JSON }}'
```

Comment:
@dottxado [mentioned](https://github.com/inpsyde/reusable-workflows/pull/180#discussion_r1975212735) something about not having to redeclare Composer authentication multiple times. Does it make sense to check if only setting it once would work? If so, please also update `build-plugin-archive.yml`.
Thank you!

Reply:
I never used that one. So probably something @Biont could check out for us. But on the other hand..it doesn't harm ;)